cli/pack/opts.go

```diff
@@ -49,7 +49,7 @@ func initAppsInfo(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx, packCtx *P
 	}
 	packCtx.AppList = appList
 	packCtx.AppsInfo, err = running.CollectInstancesForApps(packCtx.AppList, cliOpts,
-		cmdCtx.Cli.ConfigDir, cmdCtx.Integrity, true)
+		cmdCtx.Cli.ConfigDir, cmdCtx.Integrity, false)
```

Comment:
It looks like the same approach is to be applied to both instances scripts and cluster config. So both of them are required or both of them are optional and it seems wrong in case of `tt pack`. I'm not sure about cluster config, maybe it's correct that we treat it as configuration and thus able to drop from package, but it seems we are not able to treat instances scripts in the same because they contains business logic and thus look like the essential part of the package.

Reply:
And regardless of our ability to drop some part of package while preparing it, it looks quite weird that content of the package depends on current availability of some items. What if next time one pack the same application but etcd/tcs config is available? As the content is changed we will get another package for the same application. Do we expect such a behavior?